build.xml

```diff
@@ -239,6 +237,9 @@
 				<include name="config/penalties/**" />
 				<include name="config/default/**" />
 			</zipfileset>
+			<zipfileset dir="${version.dest.dir}" prefix="${zip.prefix}/html/" filemode="755" >
```

Comment:
Why do you put the release file in the html dir?

Reply:
Otherwise it's buried in a jar inside a jar, this way anyone can be told how to get it via http.